src/app/_components/supported-chains-section.tsx

```diff
@@ -78,6 +79,10 @@ const chains = [
     label: "World Chain",
     Icon: WorldChainIcon,
   },
+  {
+    label: "Aleph Zero",
+    Icon: AlephZeroIcon,
+  },
 ];
```

Comment:
The spacing on some of the longer, thinner logos looks off. 
I can see that each container in this ticker is equal length, but the SVGs inside them will grow to fill them and long thin ones take up more space. Maybe we can set a max width in the SVGs themselves to so they appear more evenly spaced

Reply:
resolved offline - we'll circle back